AzureAuthProvider.code-workspace

```diff
@@ -4,7 +4,10 @@
 			"path": "AzureResourceInterfaces"
 		},
 		{
-			"path": "AzureAuthProvider"
+			"path": "AzureOAuth2Provider"
+		},
+		{
+			"path": "AADAuthProvider"
```

Comment:
there is another code-workspace file that can be removed?
`/AzureResouceInterfaces/AzureAuthProvider.code-workspace`